postmodern/util.lisp

```diff
@@ -897,3 +896,11 @@ and calls > ~a
 ORDER BY ~a
 DESC LIMIT ~a;" num-calls ob limit)))
     (query sql-statement)))
+
+(defun list-installed-extensions ()
+  "Lists extensions that are already installed in the database."
+  (query "select * from pg_extension"))
+
+(defun list-available-extensions ()
+  "Lists extensions that are available to be installed in the database. Returns a list of lists where each sublist has the name of the extension, the default version, the installed version (if any) and a comment string."
+  (query (:select '* :from 'pg-available-extensions)))
```

Comment:
Slight inconsistency; why is one query a string and the other is an S-expression?

Reply:
Because I got distracted and came back with my mind thinking in s-sql instead of sql. Sigh. Will change to string. Thank you.